kong/conf_loader/parse.lua

```diff
@@ -432,6 +432,13 @@ local function check_and_parse(conf, opts)
         conf.ssl_dhparam = suite.dhparams
         conf.nginx_http_ssl_dhparam = suite.dhparams
         conf.nginx_stream_ssl_dhparam = suite.dhparams
+      else
+        conf.nginx_http_ssl_conf_command = "CipherString DEFAULT:@SECLEVEL=0"
```

Comment:
we need to do a append instead of rewrite the conf, since user could also already configured it

Reply:
Nginx does not support using a single `ssl_conf_command` to apply multiple commands.

It is acceptable to use multiple `ssl_conf_command` directives in the Nginx configuration.
But it seems that kong's conf_load cannot support this unless we do special processing for these instructions.